src/Contacts.ts

```diff
@@ -0,0 +1,211 @@
+import Client from './Client'
+import { privatePreferences } from '@xmtp/proto'
+import { EnvelopeWithMessage, buildUserPrivatePreferencesTopic } from './utils'
+
+export type AllowListPermissionType = 'allow' | 'block' | 'unknown'
```

Comment:
Per that document as well it uses `'allowed' | 'blocked' | 'unknown'`

https://github.com/orgs/xmtp/discussions/49